libpod/container_internal_common.go

```diff
@@ -50,6 +49,7 @@ import (
 	"github.com/containers/storage/pkg/lockfile"
 	stypes "github.com/containers/storage/types"
 	securejoin "github.com/cyphar/filepath-securejoin"
+	runcaa "github.com/opencontainers/runc/libcontainer/apparmor"
```

Comment:
I would rather keep the name apparmor, I think that is easier for future developers to understand.